cp-attendance-hybrid/src/modules/profile/profile.page.tsx

```diff
@@ -61,15 +61,15 @@ export const Profile: React.FC<ProfileScreenProps> = ({ navigation }) => {
           <EditModalItemTitle>{'Nome: '}</EditModalItemTitle>
           <EditModalItemText>{user.user?.username}</EditModalItemText>
         </EditModalItem>
-        <EditModalItem>
+        {/* <EditModalItem>
           <EditModalItemTitle>{'Telefone: '}</EditModalItemTitle>
           <EditModalItemText>{'(99)99999-9999'}</EditModalItemText>
-        </EditModalItem>
+        </EditModalItem> */}
         <EditModalItem>
           <EditModalItemTitle>{'Email: '}</EditModalItemTitle>
           <EditModalItemText>{user.user?.email}</EditModalItemText>
         </EditModalItem>
-        <ModalButton text="Editar" />
+        {/* <ModalButton text="Editar" /> */}
```

Comment:
Tem algum motivo pra estar comentado e não deletado? Se for o caso deixaria um TODO pra indicar qnd descomentar/deletar/sei lá